src/pages/trade-finance.mdx

```diff
@@ -45,25 +46,29 @@ Fetch Bill Of Lading document.
 
 Fetch Bill Of Lading document matching all the input parameters.
 
-The API will allow the caller to download the First Matching `Bill Of Lading` document matching all the input parameters.
```

Comment:
https://github.com/freight-trust/webapp/pull/159/commits/cc82f60beec26411921f2ba317f1b1f8aaddc188#diff-feb15bba8ee0f44abde8b3225058177aL48-R61

Update docs to reflect new naming conventions